tools/ubuntu-setup/README.md

```diff
@@ -27,7 +27,7 @@ Follow instructions [tools/db/README.md](../db/README.md) on how to configure a
 
   ```
   cd <home_openwhisk>
-  ./gradlew distDocker
+  sudo ./gradlew distDocker
```

Comment:
i don't think we should make this change - building and deploying in a local environment should not require privilege escalation.